content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -729,6 +731,420 @@ Example: `node.cloudprovider.kubernetes.io/shutdown: "NoSchedule"`
 
 If a Node is in a cloud provider specified shutdown state, the Node gets tainted accordingly with `node.cloudprovider.kubernetes.io/shutdown` and the taint effect of `NoSchedule`.
 
+### service.beta.kubernetes.io/aws-load-balancer-access-log-emit-interval (deprecated) {#service-beta-kubernetes-io-aws-load-balancer-access-log-emit-interval}
```

Comment:
I like the movement of these annotations, and I think it warrants a PR dedicated to this.

Reply:
That will be https://github.com/kubernetes/website/pull/38551